src/commentcnv.l

```diff
@@ -1214,7 +1233,12 @@ void convertCppComments(const BufStr &inBuf,BufStr &outBuf,const QCString &fileN
   clearCommentStack(yyscanner);
   yyextra->vhdl = FALSE;
 
-  DebugLex debugLex(Debug::Lex_commentcnv,__FILE__, qPrint(fileName));
+  if (Debug::isFlagSet(Debug::CommentCnv))
```

Comment:
did you mean to remove the DebugLex line here?

Reply:
No I didn't have the intention to remove that line, during my debugging I wanted to see the input of commentcnv and forgot to revert this.

I reverted it now.